code-challenges.js

```diff
@@ -30,6 +56,20 @@ var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "tem
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+// function called sorter takes array and returns numbers least to greatest
+const sorter = (array) => {
+  // variable called arrayCleaner that removes all non Number array values
+      //revised to cleaned array
+  let cleanedArray = array.filter(value => typeof value === "number");
+  // variable named cleanArray that has a ONLY Number values
+// console.log(cleanedArray);
+// variable finalDataArray that takes an array with numbers and returns them from least to greatest
+  let finalDataArray = cleanedArray.sort((value1, value2) => value1 - value2)
+  return finalDataArray
+}
```

Comment:
great job! you could shorten this by chaining these methods but i can see clearly what your intent was here so thats wonderful!